src/app/review-rating/review-rating.component.html

```diff
@@ -0,0 +1,43 @@
+<ion-content class="review-rating" ng-controller="reviewRatingCtrl">
+	<ng-container>		
```

Comment:
We've decided to indent with 2 spaces (not tabs), please change this file's indentation